src/app/components/feed/SaveFeed.js

```diff
@@ -453,16 +476,16 @@ const SaveFeed = (props) => {
         <div className={styles.saveFeedCard}>
           <FeedDataPoints
             readOnly={Boolean(feed.id)}
-            dataPoints={dataPoints}
-            onChange={setDataPoints}
+            dataPoints={formData.dataPoints}
+            onChange={handleSetDataPoints}
```

Comment:
I used a handler function to avoid making changes to additional files.  Originally updates only took in a value, the new Update function takes a key and value, so I needed a handler to bridge that gap.